packages/http-client-java/emitter/src/common/client.ts

```diff
@@ -19,6 +24,9 @@ export class Client extends Aspect implements Client {
 
     this.operationGroups = [];
     this.security = new Security(false);
+    this.subClients = [];
+    this.publicBuilder = true;
+    this.publicParentAccessor = false;
```

Comment:
Will the properties be set during `processClients` in code-model-builder (seems I did not find) or it's just set here as the default is enough for current cases?

Reply:
It is set in `addSubClient(..)` (a few lines below)

Reply:
Thanks, I saw `addSubClient`. I previously mean I did not find the place where we take TCGC'`s access` value and set to the property values. And now I saw your reply to Xiaofei, currently we just use Java's convention, I'm good with it.

Reply:
Yeah, last time I checked, their client access method always be access=internal, so we cannot use value from TCGC at all.